fftvis/tests/test_compare_matvis.py

```diff
@@ -120,3 +120,42 @@ def test_simulate():
 
     # Check that the results are the same
     assert np.allclose(mvis, fvis, atol=1e-5)
+
+
+def test_simulate_non_coplanar():
+    # Simulation parameters
+    ntimes = 10
+    nfreqs = 5
+    nants = 3
+    nsrcs = 20
+
+    # Set random set
+    np.random.seed(42)
```

Comment:
this way of setting random seeds is deprecated in numpy 2. Use `rng = np.random.default_rng(42)` then `rng.uniform(size=...)` later on.

Reply:
Interesting, I was not aware of that. Thanks for letting me know! Fixed that here